classes/01class/exercises/c01-aws01/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,294 @@
+                    ],
+                    "Ipv6Addresses": [],
+                    "MacAddress": "06:80:55:73:55:a2",
+                    "NetworkInterfaceId": "eni-00f1caa5876878a75",
+                    "OwnerId": "149613515908",
+                    "PrivateDnsName": "ip-172-31-35-234.ap-southeast-2.compute.internal",
+                    "PrivateIpAddress": "172.31.35.234",
+                    "PrivateIpAddresses": [
+                        {
+                            "Primary": true,
+                            "PrivateDnsName": "ip-172-31-35-234.ap-southeast-2.compute.internal",
+                            "PrivateIpAddress": "172.31.35.234"
+                        }
+                    ],
+                    "SourceDestCheck": true,
+                    "Status": "in-use",
+                    "SubnetId": "subnet-9da698eb",
+                    "VpcId": "vpc-8ad99fee",
+                    "InterfaceType": "interface"
+                }
+            ],
+            "RootDeviceName": "/dev/xvda",
+            "RootDeviceType": "ebs",
+            "SecurityGroups": [
+                {
+                    "GroupName": "default",
+                    "GroupId": "sg-7d59bd1b"
+                }
+            ],
+            "SourceDestCheck": true,
+            "StateReason": {
+                "Code": "pending",
+                "Message": "pending"
+            },
+            "Tags": [
+                {
+                    "Key": "Name",
+                    "Value": "DevOps Academy"
+                }
+            ],
+            "VirtualizationType": "hvm",
+            "CpuOptions": {
+                "CoreCount": 1,
+                "ThreadsPerCore": 1
+            },
+            "CapacityReservationSpecification": {
+                "CapacityReservationPreference": "open"
+            },
+            "MetadataOptions": {
+                "State": "pending",
+                "HttpTokens": "optional",
+                "HttpPutResponseHopLimit": 1,
+                "HttpEndpoint": "enabled"
+            }
+        }
+    ],
+    "OwnerId": "149613515908",
+    "ReservationId": "r-0c2aac62f97656013"
+}
+```
+
+- Commands to connect to the first EC2 instance:
+```
+$ ssh -i aws-marcos.pem ec2-user@13.236.195.206
+
+The authenticity of host '13.236.195.206 (13.236.195.206)' can't be established.
+ECDSA key fingerprint is SHA256:uEur5r8wVnf5oFmiSobITQZik8ArzhBF9VFj86BXo3c.
+Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
+Warning: Permanently added '13.236.195.206' (ECDSA) to the list of known hosts.
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+4 package(s) needed for security, out of 8 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-35-234 ~]$
+```
+
+- Commands to create the second EC2 instance and any additional resource required:
+```
+$ aws ec2 run-instances \
```

Comment:
same as above

Reply:
- Added a second inbound rule to the SG allowing SSH connections from the private IP of the first EC2 (bastion)

- Used SSH agent forwarding to avoid copying private key to first instance